pkg/kubelet/cm/topologymanager/fake_topology_manager.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package topologymanager
 
 import (
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Seems like an accident

Reply:
grr, updated my goimports to run with `-format-only`, I hope it will help to avoid all these unneeded changes
done